Makefile

```diff
@@ -95,20 +94,14 @@ install-shipper: build/shipper.image.$(IMAGE_TAG) build/shipper.deployment.$(IMA
 install-shipper-state-metrics: build/shipper-state-metrics.image.$(IMAGE_TAG) build/shipper-state-metrics.deployment.$(IMAGE_TAG).yaml
 	$(KUBECTL) apply -f build/shipper-state-metrics.deployment.$(IMAGE_TAG).yaml
 
-# Install a helm chart repository server with test charts. This is useful
-# mostly for end-to-end tests.
-install-helm: build/helm.image.$(IMAGE_TAG)
-	$(KUBECTL) apply -f ci/helm.service.yaml
-	sed s=\<IMAGE\>=$(shell cat build/helm.image.$(IMAGE_TAG))= ci/helm.deployment.yaml | $(KUBECTL) apply -f -
-
 # Run all end-to-end tests. It does all the work necessary to get the current
 # version of shipper on your working directory running in kubernetes, so just
 # running `make -j e2e` should get you up and running immediately. Do remember
 # do setup your clusters with `make setup` though.
-e2e: install install-helm build/e2e.test
+e2e: install build/e2e.test
 	./build/e2e.test --e2e --kubeconfig ~/.kube/config \
-		--testcharts http://$(shell $(KUBECTL) get service helm -o jsonpath='{.spec.clusterIP}'):8879 \
 		--appcluster $(SHIPPER_CLUSTER) \
+	    --testcharts $(TEST_HELM_REPO_URL) \
```

Comment:
The alignment here doesn't look right. Can't tell from github ui if this is the cause, nut make sure you're using tabs :)